topoflow/utils/svo_names.py

```diff
@@ -60,10 +60,10 @@ def get_svo_name_map():
     #-------------------------------------
     # Abbreviations used in snow_base.py
     #-------------------------------------
-    svo_name_map['hs']  = svo_name_map['h_snow']
-    svo_name_map['sw']  = svo_name_map['h_swe']
-    svo_name_map['mr']  = svo_name_map['SM']    ## See 'imr' below.
-    svo_name_map['cc']  = svo_name_map['Ecc']
+    # svo_name_map['hs']  = svo_name_map['h_snow']
```

Comment:
maybe remove these since they are copied in under line 83 (Abbreviations used in glacier_base.py), but key values 'mr' was renamed to 'imr'?